llvm/lib/Target/Mips/MipsSEISelLowering.cpp

```diff
@@ -1333,6 +1397,61 @@ SDValue MipsSETargetLowering::lowerMulDiv(SDValue Op, unsigned NewOpc,
   return DAG.getMergeValues(Vals, DL);
 }
 
+SDValue MipsSETargetLowering::lowerRemOrDiv(SDValue Op,
+                                            SelectionDAG &DAG) const {
+
+  unsigned Opcode = Op.getOpcode();
```

Comment:
Since it's an odd ABI to this call, I think it warrants some commentary to explain the ABI used and the rationale.